week-4/Homework/mandatory/2-writers.js

```diff
@@ -54,18 +56,35 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 
-
+writers.forEach((p) => {
+  console.log(
+    `Hi, my name is ${p.firstName}${p.lastName}. I am ${p.age} years old, and work as a ${p.occupation}.`
+  );
+});

```

Comment:
A minor point - there should be a space between firstName and lastName so that:
'Hi, my name is Virginia Woolf....' is rendered instead of:
'Hi, my name is VirginiaWoolf....'